.github/setup-node/action.yml

```diff
@@ -32,7 +32,10 @@ runs:
 
         - name: Install npm dependencies
           if: ${{ steps.cache-node_modules.outputs.cache-hit != 'true' }}
-          run: npm ci
+          run: |
+              npm ci --force || true
+              echo "LOGS!!!!!"
+              ls -1 C:\npm\cache\_logs |  xargs -I {} cat C:\npm\cache\_logs/{}
```

Comment:
Is the goal here to display logs in the action's output when there are failures? If so, I would leave this as just `npm ci` and add a new step that uploads the logs as an artifact. Something like:

```
- name: Upload npm logs as an artifact
  uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
  if: failure()
  with:
    name: npm-logs
    path: ~/.npm/_logs
```

Reply:
Yes, I'm just trying to debug. Thank you for the suggestion, definitely easier than my hacks.

Reply:
This won't display the logs inline, but when visiting the Summary view for the workflow run, the log can be downloaded for viewing (which is arguably much easier to work with).

Reply:
I think this is a good improvement to make in general to leave for any debugging that's needed going forward.

Reply:
Totally agree! Since you're working on this, feel free to apply this suggestion in your PR (#61289).